### ISSUE TITLE
Use multiline string for operationString

### ISSUE THREAD

**Issue by nanoxd** [apollographql/apollo-ios#179]
👋 

Looking at the generated code, it looks like we add an `operationString` with manual line breaks. Would you accept a pull request changing that to use Swift 4's [multiline string](https://github.com/apple/swift-evolution/blob/master/proposals/0168-multi-line-string-literals.md)?

## Current Operation String

```swift
public static let operationString = "query movies {\n __typename\n results\n ..."
```

## Multiline Operation String
```swift
public static let operationString = """
query movies {
  __typename
  results
}
"""
```

Thanks

**Comment by tspecht** [comment]
Would be great to get such a feature in especially when having many generated types, makes it easier to inspect the query while coding with it!

**Comment by martijnwalraven** [comment]
Yep, that makes sense now that we're using Swift 4, so a PR would definitely be welcome!

**Comment by nanoxd** [comment]
Great! I'll send a PR for that sometime today.

**Comment by designatednerd** [comment]
Looks like a PR was never opened for this but I'm definitely interested in doing this for the sake of my own sanity.

**Comment by designatednerd** [comment]
Hello! [Tooling PR 1441](https://github.com/apollographql/apollo-tooling/pull/1441) switches `operationDefinition` and `fragmentDefinition` to use multi-line string literals. Please head over there and let me know if there's anything I missed!

**Comment by designatednerd** [comment]
So there are some improvements to this which will go out with `0.14.0` - there's now a comment on the generated code which will show what's going out. The problem at the moment is that swift multiline strings and GraphQL multiline strings both use the same triple-`"` delimiter, and there's not a super elegant way around that in typescript. 

I'm going to look at using some of the new(ish) string literal stuff on this when I do some of the Swift Codegen to see if there's a way to preserve literals without tying ourselves in knots.

**Comment by designatednerd** [comment]
This shipped with non-swift codegen as part of `0.17.0`! I still plan to include this in Swift codegen, but now it's a "support the same features as legacy" rather than a "new hotness" feature, so I'll close this issue out. 😄
